docs/en/observability/synthetics-role-write.asciidoc

```diff
@@ -83,8 +83,8 @@ Additional privileges will depend on the factors below.
 [discrete]
 == If using Private Locations
 
-If the user should be able to create and update monitors hosted on
-<<synthetics-private-location,Private Locations>>, add the following privileges:
+No additional permissions are needed if the user should be able to create and update monitors hosted on
+<<synthetics-private-location,Private Locations>>. However when the agent policy is initially created within {fleet}, the following privileges are required:
```

Comment:
This is confusing to me. Is this correct? 

>The user who initially sets up a new Private Location needs additional privileges. Users who create or update monitors hosted on that Private Location do not need any additional permissions.
>
>The user who is setting up a new Private Location will need the following privileges when creating the agent policy in {fleet}:

Reply:
@colleenmcginnis kind of, but there is a nuance, but don’t know if it complicates the message.  To create the Agent Policy (under Fleet), the additional permissions are needed.  However, once it exists, the user doesn’t need the Fleet permissions to add the Private Location (under Synthetics > Settings).

Reply:
@paulb-elastic 
we only need fleet all permissions to create an agent policy, we can also remove 

```
| {kibana-ref}/kibana-privileges.html[Kibana]
| `Integrations`: `All`
| Access to Integrations in {kib}.
```

section

Reply:
@colleenmcginnis + @shahzad31, I've updated with both your feedback